web/src/main/java/no/nav/sbl/dialogarena/modiabrukerdialog/web/rest/person/PersonController.kt

```diff
@@ -134,6 +137,36 @@ class PersonController @Autowired constructor(
         val sammensatt = listOfNotNull(fornavn, mellomnavn, etternavn).joinToString(" ")
     }
 
+    data class ForeldreansvarDTO(
+        val ansvar: String,
+        val ansvarlig: String?,
+        val ansvarligUtenIdentifikator: RelatertBiPerson
```

Comment:
Om jeg forstår pdl-docen rett så vil enten `ansvarlig` eller `ansvarligUtenIdentifikator` være satt. Hvis `ansvarlig` derfor er satt så tror jeg vi må gjøre ett nytt kall til pdl for å hente ut relevant informasjon om identen. 
https://navikt.github.io/pdl/#_foreldreansvar